quedex_api/user_stream.py

```diff
@@ -185,6 +261,22 @@ class UserStream(object):
   class - see their comments for the format of the data.
   """
 
+  class BatchMode(Enum):
+    STANDARD = 1
+    TIME_TRIGGERED_CREATE = 2
+    TIME_TRIGGERED_UPDATE = 3
+
+  type_listener  = {
```

Comment:
Variable name could be better. Not sure what this is.

Reply:
I'd say this is unnecessary. Please remove this.

Reply:
Removed.